### PR TITLE
Fido2

### PR DESCRIPTION
**Pull request by merlokk**
FIDO2 commands

Next will be work with FIDO2 PIN functionality

### CODE REVIEW

File: client/Makefile [hunk 1]
```diff
@@ -23,8 +23,12 @@ JANSSONLIBPATH = ./jansson
 JANSSONLIB = $(JANSSONLIBPATH)/libjansson.a
 MBEDTLSLIBPATH = ../common/mbedtls
 MBEDTLSLIB = $(MBEDTLSLIBPATH)/libmbedtls.a
+CBORLIBPATH = ./tinycbor
+CBORLIB = $(CBORLIBPATH)/tinycbor.a
+LIBS = -I../zlib -I../uart -I../liblua -I$(MBEDTLSLIBPATH) -I$(JANSSONLIBPATH) -I$(CBORLIBPATH)
```
Comment (pwpiwi) [inline client/Makefile]: suggest `LIBINCLUDES` instead. `LIBS` would be expected on a linker command line

File: client/tinycbor/cbor.h [hunk 2]
```diff
@@ -0,0 +1,606 @@
+/****************************************************************************
+**
+** Copyright (C) 2017 Intel Corporation
```
Comment (pwpiwi) [inline client/tinycbor/cbor.h]: Which kind of license is this? A stripped down BSD license?

File: client/tinycbor/cbor.h [hunk 3]
```diff
@@ -0,0 +1,606 @@
...
+    return cbor_value_to_pretty_advance_flags(out, &copy, CborPrettyDefaultFlags);
+}
+#endif /* __STDC_HOSTED__ check */
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif /* CBOR_H */
+
```
Comment (pwpiwi) [inline client/tinycbor/cbor.h]: What is all this CBOR stuff about? What can we do with it?

File: client/tinycbor/cbor.h (hunk 2 above)
Comment (merlokk) [reply client/tinycbor/cbor.h]: https://github.com/intel/tinycbor/blob/master/LICENSE
MIT License

Comment (merlokk) [reply client/tinycbor/cbor.h]: Permissions:
 Commercial use
 Modification
 Distribution
 Private use

File: client/tinycbor/cbor.h (hunk 3 above)
Comment (merlokk) [reply client/tinycbor/cbor.h]: cbor - binary analog of json.
It needs for fido2.
I tried to write it from scratch, but i was need almost all functionality. that is why i included lib.

cbor may be used as binary json in all areas where json is used

**Pull request merged**